Dockerfile

```diff
@@ -43,19 +46,16 @@ RUN --mount=type=cache,target=/root/.cache/go-build --mount=type=cache,target=/g
 ARG GOIMPORTS_VERSION
 RUN --mount=type=cache,target=/root/.cache/go-build --mount=type=cache,target=/go/pkg go install golang.org/x/tools/cmd/goimports@${GOIMPORTS_VERSION} \
 	&& mv /go/bin/goimports /bin/goimports
-ARG DEEPCOPY_VERSION
-RUN --mount=type=cache,target=/root/.cache/go-build --mount=type=cache,target=/go/pkg go install github.com/siderolabs/deep-copy@${DEEPCOPY_VERSION} \
-	&& mv /go/bin/deep-copy /bin/deep-copy
 
 # tools and sources
 FROM tools AS base
 WORKDIR /src
-COPY ./go.mod .
-COPY ./go.sum .
-RUN --mount=type=cache,target=/go/pkg go mod download
-RUN --mount=type=cache,target=/go/pkg go mod verify
 COPY ./cmd ./cmd
 COPY ./internal ./internal
+COPY ./go.mod ./go.mod
```

Comment:
this is a bad change - this way `go mod download` is executed every time source changes, it was correct order before

Reply:
ah, right. Ok, let me fix that